google-api-grpc/proto-google-cloud-monitoring-v3/src/main/java/com/google/monitoring/v3/UptimeCheckConfigOrBuilder.java

```diff
@@ -434,6 +434,21 @@
    */
   int getSelectedRegionsValue(int index);
 
+  /**
+   *
+   *
+   * <pre>
+   * If this is `true`, then checks are made only from the 'internal_checkers'.
+   * If it is `false`, then checks are made only from the 'selected_regions'.
+   * It is an error to provide 'selected_regions' when is_internal is `true`,
+   * or to provide 'internal_checkers' when is_internal is `false`.
+   * </pre>
+   *
+   * <code>bool is_internal = 15 [deprecated = true];</code>
+   */
+  @java.lang.Deprecated
```

Comment:
@chingor13 Does it seem odd this PR added a deprecated field? I noticed this because this is tripping up `clirr-maven-plugin` when I was doing the split